components/actionbutton/index.css

```
@@ -0,0 +1,214 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+@import "../commons/index.css";
+@import "../commons/basebutton.css";
+
+/*
+small: 5x5px hold icon, 3px from the edge
+medium: 5x5px hold icon, 4px from the edge
+large: 6x6px hold icon, 5px from the edge
+extra-large: 7x7pm hold icon, 6px from the edge
+*/
+
+.spectrum-ActionButton--sizeM {
```

This simple change would avoid breaking changes by making medium variables default (and thus not requiring `spectrum-ActionButton--sizeM` to render a medium Action Button):

```suggestion
.spectrum-ActionButton,
.spectrum-ActionButton--sizeM {
```